test/redfish-core/lib/power_subsystem_test.cpp

```diff
@@ -1,45 +0,0 @@
-#include "async_resp.hpp"
```

Comment:
What in the test fails with your code?

Reply:
The test exits due to a SIGSEGV:

```
  =================================== 42/43 ====================================
  test:         power_subsystem_test
  start time:   23:25:27
  duration:     0.01s
  result:       killed by signal 11 SIGSEGV
  command:
  UBSAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1:print_stacktrace=1
  ASAN_OPTIONS=halt_on_error=1:abort_on_error=1:print_summary=1                                                                                                                                                                               MALLOC_PERTURB_=242
  /home/shawnmm/ci_test_area/bmcweb/build/power_subsystem_test
  ----------------------------------- stdout -----------------------------------
  Running main() from /googletest-1.14.0/googletest/src/gtest_main.cc
  [==========] Running 1 test from 1 test suite.
  [----------] Global test environment set-up.
  [----------] 1 test from PowerSubsystemCollectionTest
  [ RUN      ]                                                                                                                                                                                                                                PowerSubsystemCollectionTest.PowerSubsystemCollectionStaticAttributesAreExpected
  ==============================================================================
```

We discussed this test over Slack in the last day or so.  It is a very trivial test, and it dies due to this signal when I added the new properties.  You had suggested I just delete it again like we did in 1050.

Reply:
It is trivial. Sure, it can be fixed next time.